src/main/java/domain/Lotto.java

```diff
@@ -0,0 +1,80 @@
+package domain;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class Lotto {
+    private final List<Integer> numbers;
+    private Place place;
+    public Lotto(List<Integer> numbers) {
+        validate(numbers);
+        replicated(numbers);
+        this.numbers = numbers;
+    }
+
+    private void validate(List<Integer> numbers) {
+        if (numbers.size() != 6) {
+            throw new IllegalArgumentException();
+        }
+    }
+
+    @Override
+    public int hashCode() {
+        return numbers.hashCode();
+    }
+
+    @Override
+    public boolean equals(Object obj) {
+        if(obj == this){
+            return true;
+        }
+        Lotto other = (Lotto) obj;
+        return other.numbers == this.numbers;
+    }
+
+    private void replicated(List<Integer> numbers){
+        Set<Integer> set = numbers.stream()
+                .collect(Collectors.toSet());
```

Comment:
컬렉션들은 인자로 다른 컬렉션을 받아 해당 값을 사용한 컬렉션을 만드는 것이 가능합니다!!
```suggestion
        Set<Integer> set = new HashSet<>(numbers);
```